### DISCUSSION TITLE
Does anyone know how computation requirement scales with an increase in dataset size?

### DISCUSSION THREAD

**Discussion by seunggs** [Lightning-AI/pytorch-lightning#6453] [General]
I couldn't find a discussion about how computation requirement scales with an increase in dataset size. Can anyone shed some light on whether it tends to be linear or not assuming everything is equal?

Thanks!

**Comment by SkafteNicki** [comment]
By computational requirement do you mean training time or do you mean resources (gpu vram)?

**Reply by seunggs** [reply]
Any thoughts on this @SkafteNicki?

**Comment by seunggs** [comment]
I mean training time - for example, something like OpenAI's petaflop-days.
